Makefile

```diff
@@ -203,3 +205,8 @@ check:
 
 minimal :
 	PYODIDE_PACKAGES="micropip" make
+
+debug :
+	EXTRA_CFLAGS="-D DEBUG_F -s ASSERTIONS=2" \
```

Comment:
I think `ASSERTIONS=2` should be an ldflag. It affects the js code gen

Reply:
Funny, I just noticed that as I was working on #1126.